fea-rs/src/compile/compile_ctx.rs

```diff
@@ -2140,6 +2119,43 @@ impl<'a, F: FeatureProvider, V: VariationInfo> CompilationCtx<'a, F, V> {
     }
 }
 
+// testing and free fn's, friends forever
+fn resolve_name_spec(node: &typed::NameSpec) -> super::tables::NameSpec {
+    const WIN_DEFAULT_IDS: (u16, u16) = (1, 0x0409);
+    const MAC_DEFAULT_IDS: (u16, u16) = (0, 0);
+
+    let platform_id = node
+        .platform_id()
+        .map(|n| n.parse().unwrap())
+        .unwrap_or(tags::WIN_PLATFORM_ID);
+
+    let (encoding_id, language_id) = match node.platform_and_language_ids() {
+        Some((platform, language)) => (platform.parse().unwrap(), language.parse().unwrap()),
+        None => match platform_id {
+            tags::MAC_PLATFORM_ID => MAC_DEFAULT_IDS,
+            tags::WIN_PLATFORM_ID => WIN_DEFAULT_IDS,
+            _ => panic!("missed validation"),
+        },
+    };
+
+    // Drop wrapping quotes around names if present, it confuses subsequent processing of string
+    let string = match &node.string().text {
+        quoted if quoted.starts_with('"') && quoted.ends_with('"') && quoted.len() > 1 => quoted
+            .strip_prefix('"')
+            .unwrap()
+            .strip_suffix('"')
+            .unwrap()
+            .into(),
```

Comment:
`strip_prefix` is a very useful method in some cases, (specifically when you want to branch only if a prefix was present) but in this case you can just use `trim_matches('"')`.

Reply:
I want to remove only the starting/ending " which is not quite the same as trim_matches